docker-compose.yml

```diff
@@ -411,6 +412,7 @@ services:
   zkevm-mock-l1-network:
     container_name: zkevm-mock-l1-network
     image: snapchain/geth-cdk-validium-contracts:v0.0.3
```

Comment:
You can use this Docker image if you want as well:
https://hub.docker.com/layers/0xpolygon/cdk-validium-contracts/forkId8/images/sha256-4e813e45117bf277a0f7f23215f3e40c35dd503a5e8e4877534fd4086f6eb1dc?context=explore

Reply:
Hi Stefan, thanks. Which repo branch was the Docker image you mentioned built from?
The image we used was built from [this repo branch](https://github.com/Snapchain/zkevm-contracts/tree/v5.0.1-rc.2-fork.8) in which we added some deployment scripts.

Reply:
Hi, it is either the one you have used or this one: https://github.com/0xPolygonHermez/zkevm-contracts/tree/feature/verifier-v5.0.0-rc.4-fork.8 (I cannot remember anymore to be honest :slightly_frowning_face: ).

Reply:
got it, thanks.